docs/syntax-reference.rst

```diff
@@ -478,6 +505,20 @@ Records written into the stream are not timestamp-ordered with respect to other
 Therefore, the topic partitions of the output stream may contain out-of-order records even
 if the source stream for the query is ordered by timestamp.
 
+
+In either of CREATE STREAM AS SELECT, CREATE TABLE AS SELECT or INSERT INTO statements, if you have a column name
+that is a reserved word in KSQL you can access it by using back quotes. Note that when you use back quotes the
+identifier name is case sensitive. The same applies to the field names in STRUCT type.
+In the following example we select a column with name ``Properties`` and a field from address colunm named ``select``:
```

Comment:
Add a comma: "In the following example we, ..."

Reply:
Comma after `we`?